front/app/modules/commercial/customizable_navbar/admin/containers/EditPageForm/EditPageFormNotInNavbar.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
```

Comment:
The form that opens when you go to `admin/settings/navigation/edit-page/:pageId`. Wrapper around the `PageFormWithNavbarNameField` that handles submitting the form etc. Only used for pages that are **not** currently in the navbar.